doc/developer/design/README.md

```diff
@@ -100,44 +102,60 @@ short.
 
 ### Creation
 
-1. Copy the [template](./00000000_template.md) to a new date-prefixed file in
-   `doc/developer/design` and fill it in. There is a separate template that the
-   COMPUTE team prefers: [template_compute](./00000000_template_compute.md).
-2. Submit a pull request -- this makes it easy for others to add written
-   comments.
-3. "Socialize" the design with relevant stakeholders. Typically, there is a
-   small set of people who have a vested interest in the area the design
-   touches. If it's not clear who that is, ask your TL, EM, or in #eng-general.
-4. Announce that the design doc is ready for review in #eng-announce, if you
-   think it's helpful to surface the design to a larger set of people. This
-   step is optional. We have an automation that posts new ready-for-review
-   design docs to #eng-design-docs and in many cases that will be sufficient
-   to inform any interested parties.
+1. Copy the design document template to a new date-prefixed file in
+   the design/ directory of the repository you’re working in, and fill it in.
```

Comment:
Added link to template in 0ec3ce9555. The link to design was removed, because the documentation here applies to other repos, too (e.g. in cloud we have doc/design whereas here we have doc/developer/design).